r7insight/utils.py

```diff
@@ -188,10 +188,12 @@ def __init__(self, token, region, use_tls=True, verbose=True, format=None, le_da
         self.setFormatter(format)
 
         self.setLevel(logging.DEBUG)
-        if use_tls and ssl_enabled:
-            self._thread = TLSSocketAppender(verbose=verbose, le_data=le_data, le_port=le_port, le_tls_port=le_tls_port)
-        else:
+
+        if not use_tls or (not ssl_supported and allow_plaintext_fallback):
```

Comment:
Are we sure we want falling back to plaintext configurable?

Reply:
https://issues.corp.rapid7.com/browse/LOG-10916
Yup